lib/scala/project-manager/src/test/scala/org/enso/projectmanager/BaseServerSpec.scala

```diff
@@ -203,7 +204,7 @@ class BaseServerSpec extends JsonRpcServerTestKit with BeforeAndAfterAll {
   /** Tests can override this value to request a specific engine version to be
     * preinstalled when running the suite.
     */
-  val engineToInstall: Option[SemVer] = None
+  val engineToInstall: EnsoVersion = DefaultEnsoVersion
```

Comment:
Why this change?

I think the new wording is not in-line with what this variable is responsible for.

The pre-installed version is used to ensure that some (dummy) version is installed in the test environment so that tests that require it will work as expected.

If no version is preinstalled (formerly `engineToInstall = None`), it just means that no versions are available and so the operations will fail or install some default version (depending on the operation).

But I think that the case of having no preinstalled versions is not really described well by the wording of `Default`, it is not used to test the behaviour of having some default version but of having no versions.

Reply:
Just wanted the `Encoder` instance. :blush: Reverted.